frappe/database/database.py

```diff
@@ -71,29 +71,22 @@ class InvalidColumnName(frappe.ValidationError):
 	def __init__(
 		self,
 		host=None,
+		port=None,
```

Comment:
Sorry forgot to point this out earlier - better to not change the order of arguments? Port can remain at the end

Reply:
done in https://github.com/frappe/frappe/pull/24332/commits/a1a653632f4133f29e0f513aba8dd7a192a8567e :+1: